<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgExplain</class>
 <widget class="QDialog" name="DlgExplain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Explain</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="ck_verbose">
          <property name="text">
           <string>Verbose</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_analyse">
          <property name="text">
           <string>Analyse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_costs">
          <property name="text">
           <string>Costs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_buffers">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Buffers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_timing">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Timing</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Format</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rb_text">
          <property name="text">
           <string>Te&amp;xt</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_xml">
          <property name="text">
           <string>X&amp;ML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_json">
          <property name="text">
           <string>JSON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_yaml">
          <property name="text">
           <string>&amp;YAML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_graph">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Graph</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgExplain</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgExplain</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
